src/main/java/org/terasology/module/health/events/RegisterRegenEvent.java

```diff
@@ -0,0 +1,17 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.module.health.events;
+
+public class RegisterRegenEvent {
```

Comment:
where is the regen registation processed? :thinking:

Reply:
nowhere yet. see commit message:

> This prepares to switch to a model and event flow better suited for the ECS and module collaboration for the regeneration feature. Both events are not used yet (neither sent nor reacted on).